Ошибка: HTTPConnectionPool(host='71', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000002027AAE4C70>: Failed to resolve '71' ([Errno 11001] getaddrinfo failed)"))